### DISCUSSION TITLE
API Data 규격

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#10] [Ideas]
## Issue
- Id : number
- Title : string
- Body : string
- Label : [{label}, {label}, {}, ...]
- Milestone : {milestone}
- Assignee : [{user}, {user}, ...]
- Author : {user}
- Date : "YYYY-MM-DD : HH - mm"
- Comments : [{comment}, {comment}, ...]
- Status : boolean
## Label
- id : number
- name : string
- body : string
- color_code : "#NNNNNN"
## Milestone
- Id
- Title
- Body
- Time : "YYYY-MM-DD : HH - mm"
- Issues : [{issue}, {issue}, ..]
## Comment
- Id
- Author : {user}
- Body
- Time : "YYYY-MM-DD : HH - mm"
## User
- token ?
- name
- image : src, string

**Comment by JacksonPk** [comment]
Date 형식 = "YYYY-MM-DD : HH - mm" 으로 부탁드립니다.
